CHANGELOG.rst

```diff
@@ -17,6 +17,12 @@ Important Changes
 Other
 -----
 
+* Molecule docker images will use the following convention on tags going forwards:
+  * ``latest``: corresponds to the master branch, which should be viewed as unstable
```

Comment:
seems wrong: latest should point to last release (not last pre-release)
master should point to last build from master, what counts as unstable

Reply:
Not currently possible, hence suggesting people use named versions. We will look at fixing this in 2.21